### PR TITLE
GUI/menus/add scroll/172

### PR DESCRIPTION
**Pull request by osvegn**
Implement scroll in the bottom menu for players and eggs.

### CODE REVIEW

File: src/zappy_gui_src/Core/SFML/Interface/ServerCommunication/ServerCommunication.hpp [hunk 1]
```diff
@@ -98,6 +98,8 @@ namespace gui {
             /// \note Type value available are: `player`, `egg`, `tile`.
             /// \param type The type of the entity.
             void removeEntities(std::string &type);
+
+            void scroll(const float &scrollValue);
```
Comment (LucasTesnier) [inline src/zappy_gui_src/Core/SFML/Interface/ServerCommunication/ServerCommunication.hpp:102]: Missing documentation

File: src/zappy_gui_src/Core/SFML/WrapperSFML/Event.hpp [hunk 2]
```diff
@@ -129,7 +129,18 @@ class Event {
         inline sf::Mouse::Button getButton() const {
             return _button;
         };
+
+        inline void setMouseScrollValue(float value) {
```
Comment (LucasTesnier) [inline src/zappy_gui_src/Core/SFML/WrapperSFML/Event.hpp:133]: Missing doc

File: src/zappy_gui_src/Core/SFML/WrapperSFML/Event.hpp [hunk 3]
```diff
@@ -129,7 +129,18 @@ class Event {
...
+            _scrollValue = value;
+        };
+
+        inline const float &getMouseScrollValue() const {
```
Comment (LucasTesnier) [inline src/zappy_gui_src/Core/SFML/WrapperSFML/Event.hpp:137]: Missing doc

**Review by LucasTesnier — CHANGES_REQUESTED** [review]
Some missing documentation, otherwise good job

**Review by LucasTesnier — APPROVED** [review]
NTR!

**Review by pariciadra — COMMENTED** [review]
LGTM maybe except the 200 char of compute, nevermind

**Pull request merged**